webpack/assets/javascripts/react_app/constants.js

```diff
@@ -19,6 +19,7 @@ export const getControllerSearchProps = (
     useKeyShortcuts: true,
   },
   bookmarks: {
+    id,
```

Comment:
should it be called `autocompleteID` ?

Reply:
Not sure, it's also used for the id of the bookmarks modal